assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/workers/bootstrap-configs.tf

```diff
@@ -0,0 +1,39 @@
+locals {
+  worker_bootstrap_tokens = [
```

Comment:
We should have one token per worker pool, so all instances in autoscaling group use exactly same configuration.